### ISSUE TITLE
Make run python script function generic

### ISSUE THREAD

**Issue by vladNed** [oxide-tech/jonescli#28]
## Reasoning

There are multiple functions for running a python script, found in `joneslib/loader.rs` file. These methods should instead be merged into one method and given a script (as a string) run only once. Maybe also inline this for better performance.

## Implementation

Ideas:
- make separate python scripts with possibility to pass arguments to it, such as the file/dir path.
- make separate python scripts but also compile them so its much easier to deliver with the jones package itself (should be tested locally first)
- move the scripts to a different file and read them from there. Can either be a python script or it can be a rust file and make the script as a static variable
